docs/spec.md

```diff
@@ -2716,8 +2716,7 @@ Produces the size of the given `dimension` of the `operand`.
 
 ### Constraints
 
-* (C1) 0 $\le$ `dimension` $\lt$ `rank(operand)`.
-  [todo](https://github.com/openxla/stablehlo/issues/790)
+  * (C1) 0 $\le$ `dimension` $\lt$ `rank(operand)`.
```

Comment:
Per https://github.com/openxla/stablehlo/blob/main/.markdownlint.yaml#L21, this shouldn't be indented.